src/main/kotlin/com/google/container/tools/diagnostics/GoogleFeedbackTask.kt

```diff
@@ -0,0 +1,153 @@
+/*
+ * Copyright 2018 Google Inc. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.google.container.tools.diagnostics
+
+import com.android.tools.idea.diagnostics.error.AnonymousFeedback
+import com.google.common.annotations.VisibleForTesting
+import com.intellij.openapi.progress.ProgressIndicator
+import com.intellij.openapi.progress.Task
+import com.intellij.openapi.project.Project
+import com.intellij.util.net.HttpConfigurable
+import java.io.IOException
+import java.net.HttpURLConnection
+
+/** Reports an error to Google Feedback in the background.  */
+class GoogleFeedbackTask @VisibleForTesting
+constructor(
```

Comment:
since we can have only one constructor, can we move this into the class level constructor instead?

Reply:
I don't think so since kotlin requires a primary constructors if one or more constructors at to be used.

Reply:
oh I misunderstood your comment. yes - will do.